packages/beachball/src/getChangedPackages.ts

```diff
@@ -72,11 +72,8 @@ export function getChangedPackages(branch: string, cwd: string, fetch: boolean)
   });
 
   if (changeFilePackageSet.size > 0) {
-    console.log(
-      `Your local repository already has change files for these packages: ${[...changeFilePackageSet]
-        .sort()
-        .join(', ')}`
-    );
+    console.log('Your local repository already has change files for these packages:' +
```

Comment:
This will insert the comma character before each line. Let ’s  join it with new line character instead maybe?